tasks/setup_registry.yaml

```diff
@@ -112,3 +112,7 @@
       --to={{ local_registry }}/{{ setup_registry.local_repo }} \
       --to-release-image={{ local_registry }}/{{ setup_registry.local_repo }}:{{ setup_registry.release_tag }}
     register: registry
+
+  - name: Mirror NFS image (ppc64le)
+    when: ppc64le
+    shell: "oc image mirror docker.io/ibmcom/nfs-client-provisioner-ppc64le:latest registry.{{ dns.clusterid }}.{{ dns.domain }}:5000/nfs-client-provisioner-ppc64le:latest -a ~/.openshift/pull-secret-updated"
```

Comment:
This should be good for now. We might want to consider making the nfs image a var (`nfs.nfs_image`), but this is good for now